tests/integration_tests/resources/dsl/scripts/workflows/test_deployment_id_parameters.py

```diff
@@ -1,11 +1,28 @@
+import json
 import sys
 
-import json
+from cloudify.decorators import workflow
+from cloudify.manager import get_rest_client
+from cloudify.workflows import ctx
 
 
+@workflow
 def test_parameter(name, value):
     assert value is not None
     print("Tested parameter '{0}' is {1}".format(name, value))
+    client = get_rest_client()
+
+    node = ctx.get_node('test_node')
+    if not node:
+        return
+    for ni in client.node_instances.list(node_id=node.id):
+        rp = ni.runtime_properties
+        rp.update({'tested': True, name: value})
+        client.node_instances.update(
+            ni.id,
+            version=ni.version,
+            runtime_properties=rp
+        )
```

Comment:
```suggestion
    node = ctx.get_node('test_node')
    if not node:
        return
    for ni in node.instances:
        rp = ni.runtime_properties
        rp.update({'tested': True, name: value})
        ctx.update_node_instance(
            ni.id,
            runtime_properties=rp,
            version=ni.version,
        )
```
In principle, let's avoid using the rest-client directly. That stuff is exposed on the workflow ctx.

(the real underlying reason is cfy local compat, which is irrelevant here, but... well, it also makes it shorter)